docs/install_linux_mac.rst

```diff
@@ -111,6 +111,23 @@ Debian/Raspbian Buster. This guide will tell you how. First, run the following c
 
 Complete the rest of the installation by `installing Python 3.9 with pyenv <install-python-pyenv>`.
 
+---
+
+.. _install-debian11:
+
+~~~~~~~~~~~~~~~~~~~~
+Debian 11 Bullseye
+~~~~~~~~~~~~~~~~~~~~
+
+Debian 11 "Bullseye" has all required packages available in official repositories. Install them
+with apt:
+
+.. prompt:: bash
+
+    sudo apt -y install python3 python3-dev python3-venv python3-pip git openjdk-11-jre-headless build-essential nano
```

Comment:
Better to add `sudo apt update` in case repositories are outdated (this was a reason of failure for CI build [here](https://github.com/jack1142/Red-Install-Tests/runs/3526740476?check_suite_focus=true)):
```suggestion
    sudo apt update
    sudo apt -y install python3 python3-dev python3-venv python3-pip git openjdk-11-jre-headless build-essential nano
```

Reply:
Good point. I just auto-pilot when I do apt stuff and just do `sudo apt update` when I do package updates. Line added in!